### DISCUSSION TITLE
What's in the dev releases on PyPI?

### DISCUSSION THREAD

**Discussion by matkozak** [duckdb/duckdb#7728] [Q&A]
Hi,

I recently saw a PR that would be very useful to me and am excited to try it out when it lands in a dev release! Problem is, I see no obvious connection between the numbering of the dev releases and any of the branches, tags, commits or PRs on GitHub.

Question: how can I tell which commits any given pre-release build does and does not contain? I sort of see it's a nightly build, but from which branch?

Thanks!

**Comment by l1t1** [comment]
it seems there is  no rule on the release number of dev.
